s5/s5.go

```diff
@@ -57,6 +57,10 @@ func (s *S5) Run() error {
 		return err
 	}
 
+	if ! sqlReader.HasData() {
+		return nil
```

Comment:
If the specified query yielded no results, than creating a empty file on s3 would be the correct result. I was under the assumption though, that the underlying issue was, that we are trying to upload empty files. Therefore the whole point of the commit was to omit the upload if the result of a query yielded no results. As you stated in your mail, my assumption was wrong and I think we should drop this pull-request.